src/components/apps/launch/index.js

```diff
@@ -81,6 +81,8 @@ const Launch = ({
 
     const preferences = bootstrapInfo?.preferences;
     const notify = preferences?.enableAnalysisEmailNotification || false;
+    const notifyPeriodic = preferences?.enablePeriodicEmailNotification || true;
```

Comment:
This won't work if the user sets this preference to `false`, since the `||` here will set `notifyPeriodic` to `true` anyway.

You could coerce it into a boolean like this:
```javascript
const notifyPeriodic = !!preferences?.enablePeriodicEmailNotification;
```

Reply:
Oh, good catch, yeah. I'll make & push that change.